content/docs/practices/instrumentation.md

```diff
@@ -1,250 +1,250 @@
 ---
-title: Instrumention
+title: Instrumentation
 sort_rank: 3
 ---
 
 # Instrumentation
 
-This page gives guidelines for when you're adding instrumentation to your code.
+This page provides an opinionated set of guidelines for instrumenting your code.
 
 ## How to instrument
 
 The short answer is to instrument everything. Every library, subsystem and
-service should have at least a few metrics to give you a rough idea of how it's
+service should have at least a few metrics to give you a rough idea of how it is
 performing.
 
-Instrumentation should be an integral part of your code, instantiate the metric
+Instrumentation should be an integral part of your code. Instantiate the metric
 classes in the same file you use them. This makes going from alert to console to code
-easy when you're chasing an error.
+easy when you are chasing an error.
 
 ### The three types of services
 
-For monitoring purposes services can generally be broken down into three types,
-online serving, offline processing and batch jobs. There is overlap between
+For monitoring purposes, services can generally be broken down into three types:
+online-serving, offline-processing, and batch jobs. There is overlap between
 them, but every service tends to fit well into one of these categories.
 
-#### Online serving systems
+#### Online-serving systems
 
-An online serving system is one where someone is waiting on a response, for
-example most database and http requests fall into this category.
+An online-serving system is one where a human is expecting an immediate response.
```

Comment:
There's no requirement that there's a human, a database used by a batch system where query latency matters is an online serving system.